### ISSUE TITLE
Element tagName is not patching by id

### ISSUE THREAD

**Issue by reenko** [patrick-steele-idem/morphdom#72]
I create test:

```
it('patch tags by id', function() {
    var el1 = document.createElement('div');
    el1.innerHTML = '<span id="boo" class="foo"></span>';

    var el2 = document.createElement('div');
    el2.innerHTML = '<div id="boo"></div>';

    var morphedEl = morphdom(el1, el2);

    expect(el1.innerHTML).to.equal('<div id="boo"></div>');
});
```

Failed:

```
   AssertionError: expected '<span id="boo"></span>' to equal '<div id="boo"></div>'
      + expected - actual

      -<span id="boo"></span>
      +<div id="boo"></div>
```

**Comment by patrick-steele-idem** [comment]
Interesting. I'll take a look. Thanks for providing the test case.

**Comment by patrick-steele-idem** [comment]
New version published with fix: `morphdom@1.4.41`
